dev-libs/libpcre2/patches/libpcre2-10.43.patchset

```diff
@@ -0,0 +1,54 @@
+From 375dd8b61787952e1bf23916567a3376a62b951c Mon Sep 17 00:00:00 2001
+From: Jerome Duval <jerome.duval@gmail.com>
+Date: Tue, 30 Apr 2019 18:33:26 +0200
+Subject: gcc2 fix
```

Comment:
If this version isn't built with gcc2, it doesn't need fixes for it. Can't the newer version be also used for gcc2 then?

Reply:
Fixed.